src/main/java/de/tum/in/www1/artemis/web/rest/LectureResource.java

```diff
@@ -258,7 +262,7 @@ public ResponseEntity<Lecture> importLecture(@PathVariable long sourceLectureId,
 
         final var savedLecture = lectureImportService.importLecture(sourceLecture, destinationCourse);
         channelService.createLectureChannel(savedLecture, Optional.empty());
-
+        helpExecuteIngestionPipeline(savedLecture);
```

Comment:
I'd put this into the LectureImportService

Reply:
this is only relevant for that class, it is not relevant for others. I just added it to not have code duplicates.